django/thunderstore/permissions/models/visibility.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+
+class VisibilityFlags(models.Model):
```

Comment:
Would it make sense to add helper methods here? E.g. `deacticate()` which sets all but admin flags to false? Not necessarily in this PR, but the sooner it's done, the less change different usage practices emerges on the calling side of the code.